source/clear-linux/guides/telemetrics/telem-guide.rst

```diff
@@ -0,0 +1,566 @@
+.. _telem-guide:
+
+Telemetrics
+###########
+
+Telemetrics in |CL-ATTR| is a client and server solution used to collect data from running |CL| systems to help quickly identify and fix bugs in the OS.  Both client and server are customizable, and an API is available on the client side for instrumenting your code for debug and analysis.
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Description
+*************
+
+Telemetry, one of the key features of |CL|, enables developers to observe and proactively address issues in the OS before end users are impacted.
+
+Telemetrics is a combination word made from:
+
+* Telemetry, which is sensing and reporting data.
+* Analytics, which is using visualization and statistical inferencing to make sense of the reported data.
+
+|CL| telemetry reports system-level debug/crash information using specialized probes. The probes monitor system tasks such as swupd, kernel oops, machine error checks, and the BIOS error report table for unhandled hardware failures. Telemetry enables real-time issue reporting to allow system developers to quickly focus on an issue and monitor corrective actions.
+
+|CL| telemetry is fully customizable and can be used during software development for debugging purposes. You can use the  libtelemetry library in your code to create custom telemetry records. You can also use the telem-record-gen utility in script files for light touch record creation where instrumenting code files doesn't make sense.
+
+Architecture
+============
+
+|CL| telemetry has two fundamental components, which are shown in figure 1:
+
+* Client: generates and delivers records to the backend server via the network
+
+* Backend: receives records sent from the client and displays the cumulative content through a specialized web interface.
+
+.. figure:: ../../concepts/figures/telemetry-about-1.png
```

Comment:
Yep -- I expect we'll update all the images, I linked to ones currently live so that they'd show up. One of the open questions is what/how we transition the other guide and tutorial info so we have it all in one place.

Reply:
added updated image